srtp.go

```diff
@@ -30,9 +30,19 @@ func (c *Context) decryptRTP(dst, ciphertext []byte, header *rtp.Header, headerL
 		}
 	}
 
-	dst = growBufferSize(dst, len(ciphertext)-authTagLen)
+	cipher := c.cipher
+	if len(c.mkis) > 0 {
```

Comment:
As the default master key is appended to `c.mkis`, this is always true even if `MasterKeyIndicator` option is not specified. `*1`
https://github.com/pion/srtp/blob/4bbc17f9768e73f06e8178e2fe16908d138b9b40/context.go#L93-L97

Reply:
Fixed context creation, it should not add empty MKI to mki map.